libpas/src/libpas/filc_runtime.c

```diff
@@ -7748,6 +7749,13 @@ int filc_native_zsys_flock(filc_thread* my_thread, int fd, int operation)
     return result;
 }
 
+long filc_native_zsys_times(filc_thread* my_thread, filc_ptr times_ptr)
+{
+    if (filc_ptr_ptr(times_ptr))
+        filc_check_read(times_ptr, sizeof(struct tms));
```

Comment:
Oh man, didn't notice this before. This needs to be filc_check_write, since the syscall writes into times_ptr.